metals/src/main/scala/scala/meta/internal/metals/StacktraceAnalyzer.scala

```diff
@@ -0,0 +1,227 @@
+package scala.meta.internal.metals
+
+import java.io.FileWriter
+import java.net.URLEncoder
+import java.util.Collections.singletonList
+
+import scala.io.Source
+import scala.util.Try
+
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.io.AbsolutePath
+
+import com.google.gson.JsonPrimitive
+import org.eclipse.lsp4j.Location
+import org.eclipse.{lsp4j => l}
+
+class StacktraceAnalyzer(
+    workspace: AbsolutePath,
+    definitionProvider: DefinitionProvider,
+    gotoicon: String,
+    commandsInHtmlSupported: Boolean
+) {
+
+  def analyzeCommand(
+      commandParams: l.ExecuteCommandParams
+  ): Option[l.ExecuteCommandParams] = {
+    parseJsonParams(commandParams)
+      .flatMap(analyzeStackTrace)
+  }
+
+  def matches(path: AbsolutePath): Boolean =
+    path == workspace.resolve(Directories.stacktrace)
+
+  def highlight(
+      path: AbsolutePath,
+      lineNumber: Int
+  ): java.util.List[l.DocumentHighlight] = {
+    val lineOpt = readStacktraceFileLine(path, lineNumber)
+    lineOpt match {
+      case Some(line) =>
+        val from = new l.Position(lineNumber, line.indexOf("at ") + 3)
```

Comment:
I don't think document highlight makes sense unless we try to highlight lines referencing the same line/file

Highlighting one line doesn't seem to add anything.

Reply:
I think it was a nice visual aspect for going to definition as you always see relation 'highlight <-> gododefinition available'
but since goto def is dropped this can be dropped too